tests/tests.bats

```diff
@@ -5,7 +5,8 @@ load 'test_helper/bats-assert/load'
 load test_helpers
 
 IMAGE="bats-opbeans"
-CONTAINER="opbeans-flask"
+CONTAINER="opbeans-python"
```

Comment:
Good catch! I'd say to keep `opbeans-flask` and change the docker-compose references to opbeans-python for `opbeasn-flask` to avoid the misleading with the `opbeans-python` project.

What do you think?

Reply:
👍